### PR TITLE
Multigraph support in edit mode

### PR DESCRIPTION
**Pull request by RazinShaikh**
I would like to split multigraph support work into several PRs and I think it is fine to merge as the main branch is already broken. This PR adds the ability to create multiple edges in the edit mode with multigraph. This PR depends on the PR https://github.com/Quantomatic/pyzx/pull/236 in pyzx which should be merged first.

### CODE REVIEW

File: zxlive/graphscene.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def __init__(self) -> None:
         self.setSceneRect(0, 0, 2*OFFSET_X, 2*OFFSET_Y)
         self.setBackgroundBrush(QBrush(QColor(255, 255, 255)))
         self.vertex_map: dict[VT, VItem] = {}
-        self.edge_map: dict[ET, EItem] = {}
+        self.edge_map: dict[tuple[ET, int], EItem] = {}
```
Comment (jvdwetering) [inline zxlive/graphscene.py:55]: Is this correct? You already redefined ET to have an additional int in it.

**Comment by jvdwetering** [comment]
This is giving 'AttributeError: 'Edge' object has no attribute 'get_edge_count'' in the test suite. Is this expected?

File: zxlive/graphscene.py (hunk 1 above)
Comment (RazinShaikh) [reply zxlive/graphscene.py:55]: The type of ET has (source, target, edge_type) but this is not enough information to determine which exact edge it is in the drawing. So I give an additional index in the edge_map.

**Comment by RazinShaikh** [comment]
I added the get_edge_count method in the PyZX PR that's now merged.
https://github.com/Quantomatic/pyzx/blob/a11c8f8a2d6a086a0b9a2cb69ab85384a1aa762a/pyzx/graph/multigraph.py#L53-L57

**Pull request merged**